serverless/src/shared/getProviderUrlsMap.js

```diff
@@ -0,0 +1,72 @@
+import { getProviderUrlsQuery } from '@/shared/operations/queries/getProviderUrlsQuery'
+import { sparqlRequest } from '@/shared/sparqlRequest'
+
+/**
+ * Fetches provider URLs and creates a map based on the given scheme.
+ * @param {string} scheme - The scheme to use for the query.
+ * @returns {Object} A map of subject values to their corresponding business object values.
+ *
+ * @example
+ * // Fetch provider URLs for the 'example' scheme
+ * const providerMap = await getProviderUrlsMap('example');
+ *
+ * // Example output:
+ * // {
+ * //   'http://example.com/provider1': ['BusinessObject1', 'BusinessObject2'],
```

Comment:
is this a map of provider ids to their corresponding urls, if so maybe just say 'URL1', 'URL2'?